etc/migration/10.3-to-11.0/migration.cql3

```diff
@@ -0,0 +1,2 @@
+ALTER TABLE "Principals" ADD "emailChange" text;
+ALTER TABLE "Principals" ADD "emailVerified" boolean;
```

Comment:
Yes because of that reason but also I thought we might want to discern between the following situations:

 - A user logged in through SSO but has no email address
 - A user updated his email address but hasn't confirmed it yet

I don't feel strongly about either approach though